pkg/certificate/manager.go

```diff
@@ -105,7 +105,10 @@ func (m *Manager) EnsureCertificate(certReq Request, ownerName string) (Certific
 	keyFile := filepath.Join(m.K0sVars.CertRootDir, fmt.Sprintf("%s.key", certReq.Name))
 	certFile := filepath.Join(m.K0sVars.CertRootDir, fmt.Sprintf("%s.crt", certReq.Name))
 
-	uid, _ := users.GetUID(ownerName)
+	uid, err := users.GetUID(ownerName)
+	if err != nil {
+		logrus.Warnf("failed to get uid for %s, using uid 0: %v", ownerName, err)
```

Comment:
Can you explicitly set uid to 0 here instead of relying on the behavior of GetUID here?